app/controllers/concerns/hydranorth/collections/base_query.rb

```diff
@@ -0,0 +1,27 @@
+module Hydranorth::Collections::BaseQuery
+  extend ActiveSupport::Concern
+  include Blacklight::Catalog
+  include Hydra::Collections::SelectsCollections
+  include Hydranorth::Permissions
+
+  def collections_search_builder_class
+    ::CollectionSearchBuilder
+  end
+
+  def access_levels
+    { read: [:read, :edit], edit: [:edit] }
+  end
+
+  def perform_collection_query(query_string, access_level)
+    # need to check for _tesim and _bsi in solr query because ActiveFedora does not allow false to be passed
+    # need to know the user if there is an access level applied otherwise we are just doing public collections
```

Comment:
This comment refers to the logic that would be in the query_string. Makes more sense if it's close to  app/controllers/concerns/hydranorth/collections/collection_selection.rb#6 and similar.